test/uk/gov/hmrc/plasticpackagingtax/returns/config/ErrorHandlerSpec.scala

```diff
@@ -51,34 +44,5 @@ class ErrorHandlerSpec
       result must include("message")
     }
 
-    "handle no active session authorisation exception" in {
-
-      val error  = new NoActiveSession("A user is not logged in") {}
-      val result = Future.successful(errorHandler.resolveError(FakeRequest(), error))
-      val expectedLocation =
-        s"""http://localhost:9949/auth-login-stub/gg-sign-in?continue=${urlEncode(
-          "http://localhost:8505/plastic-packaging-tax/returns/submit-return"
-        )}"""
-
-      status(result) mustBe SEE_OTHER
-      redirectLocation(result) mustBe Some(expectedLocation)
-    }
-
-    "handle insufficient enrolments authorisation exception" in {
-
-      val error  = InsufficientEnrolments("HMRC-PPT-ORG")
-      val result = Future.successful(errorHandler.resolveError(FakeRequest(), error))
-
-      status(result) mustBe SEE_OTHER
-      redirectLocation(result).value must endWith("/unauthorised")
-    }
-
-    "handle all exceptions" in {
```

Comment:
Do we still need this test, e.g. for 500s, etc.?

Reply:
Good question.  I think not but I've found some other redundant code in that area.

Reply:
I've added the test back.  But it is just testing that all exceptions are forwarded to the ErrorHandler's base class.